awsmt/resource_playback_configuration_test.go

```diff
@@ -1,390 +1,137 @@
 package awsmt
 
 import (
-	"fmt"
-	"github.com/aws/aws-sdk-go/service/mediatailor"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-	"regexp"
 	"testing"
 )
 
-func init() {
-	resource.AddTestSweepers("test_playback_configuration", &resource.Sweeper{
-		Name: "test_playback_configuration",
-		F: func(region string) error {
-			client, err := sharedClientForRegion(region)
-			if err != nil {
-				return fmt.Errorf("error getting client: %s", err)
-			}
-			conn := client.(*mediatailor.MediaTailor)
-			names := []string{"test_playback_configuration_awsmt", "example_tag_removal", "testacc_example_playback"}
-			for _, n := range names {
-				_, err = conn.DeletePlaybackConfiguration(&mediatailor.DeletePlaybackConfigurationInput{Name: &n})
-				if err != nil {
-					return err
-				}
-			}
-			return nil
-		},
-	})
-}
-
-func TestAccPlaybackConfigurationResourceBasic(t *testing.T) {
-	resourceName := "awsmt_playback_configuration.r1"
+func TestAccPlaybackConfigurationResource(t *testing.T) {
 	resource.Test(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t) },
-		ProviderFactories: ProviderFactories,
-		CheckDestroy:      testAccCheckPlaybackConfigurationDestroy,
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
 		Steps: []resource.TestStep{
```

Comment:
I think resources need more testing, especially when it comes to updating them

Reply:
I tested updating everything I know how to change / what can be possible inputs for it, im not fully sure how to add more tests atm.